src/main/java/uk/gov/hmcts/divorce/legaladvisor/event/LegalAdvisorMakeDecision.java

```diff
@@ -47,6 +52,7 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             .showSummary()
             .showEventNotes()
             .aboutToSubmitCallback(this::aboutToSubmit)
+            .submittedCallback(this::submitted)
```

Comment:
is there a reason notification is sent in submitted can we not send notification as part of about to submit so in case notification fails legal advisor can retrigger it ?

Reply:
Happy to move to about to submit - just thought having it there would ensure the state had been correctly set in about to submit